app/objects/secondclass/c_executor.py

```diff
@@ -0,0 +1,87 @@
+import marshmallow as ma
+
+from app.utility.base_object import BaseObject
+
+from app.objects.secondclass.c_parser import ParserSchema
+from app.objects.secondclass.c_variation import Variation, VariationSchema
+
+
+class ExecutorSchema(ma.Schema):
+    name = ma.fields.String(missing=None)
+    platform = ma.fields.String(missing=None)
+    command = ma.fields.String(missing=None)
+    code = ma.fields.String(missing=None)
+    language = ma.fields.String(missing=None)
+    build_target = ma.fields.String(missing=None)
+    payloads = ma.fields.List(ma.fields.String(), missing=[])
+    uploads = ma.fields.List(ma.fields.String(), missing=[])
+    timeout = ma.fields.Int(missing=60)
+    parsers = ma.fields.List(ma.fields.Nested(ParserSchema()), missing=[])
+    cleanup = ma.fields.List(ma.fields.String(), missing=[])
+    variations = ma.fields.List(ma.fields.Nested(VariationSchema()), missing=[])
+    additional_info = ma.fields.Dict(keys=ma.fields.String(), values=ma.fields.String())
+
+    @ma.post_load
+    def build_executor(self, data, **_):
+        return Executor(**data)
+
+
+class Executor(BaseObject):
+
+    schema = ExecutorSchema()
+    display_schema = ExecutorSchema()
+
+    RESERVED = dict(payload='#{payload}')
+
+    HOOKS = dict()
+
+    @property
+    def test(self):
+        """Get command with app property variables replaced"""
+        return self.decode_bytes(self.replace_app_props(self.encode_string(self.command)))
```

Comment:
There used to be a `@test.setter`. Is that needed?

Reply:
No, I'm keeping this in for compatibility with the areas that expect the ability to replace `app.*` variables. This should only be read in a few places.